idd/Energy+.idd.in

```diff
@@ -40246,14 +40250,12 @@ ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
         \note This field is set to zero flow when the VRF terminal unit is connected to
         \note central dedicated outdoor air through air terminal single duct mixer object.
   A5 ,  \field Supply Air Fan Operating Mode Schedule Name
-        \required-field
```

Comment:
This and next 2 fields are (will be) removed so that air loop and OA system TUs do not require a fan object.